From ccbe3369a69af38dbf8fdf3a0a1b27ba534506ea Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 11 Mar 2014 05:23:39 +0100
Subject: [PATCH] man: multiple sleep modes are to be separated by whitespace,
 not commas

As pointed out by Jason A. Donenfeld.

Conflicts:
	man/systemd-sleep.conf.xml

(cherry-picked from 252094eb05c58270a0bc35b14ad30a126ddbb3bb)

Resolves: #1147524
---
 man/systemd-sleep.conf.xml | 18 ++++++++----------
 1 file changed, 8 insertions(+), 10 deletions(-)

diff --git a/man/systemd-sleep.conf.xml b/man/systemd-sleep.conf.xml
index dc4b0da..d0ea6d8 100644
--- a/man/systemd-sleep.conf.xml
+++ b/man/systemd-sleep.conf.xml
@@ -127,10 +127,10 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
         <citerefentry><refentrytitle>systemd-suspend.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
         <citerefentry><refentrytitle>systemd-hibernate.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>, or
         <citerefentry><refentrytitle>systemd-hybrid-sleep.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>.
-        More than one value can be specified by seperating
-        multiple values with commas. They will be tried
+        More than one value can be specified by separating
+        multiple values with whitespace. They will be tried
         in turn, until one is written without error. If
-        neither suceeds, the operation will be aborted.
+        neither succeeds, the operation will be aborted.
         </para></listitem>
       </varlistentry>
 
@@ -145,10 +145,10 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
         <citerefentry><refentrytitle>systemd-suspend.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
         <citerefentry><refentrytitle>systemd-hibernate.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>, or
         <citerefentry><refentrytitle>systemd-hybrid-sleep.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>.
-        More than one value can be specified by seperating
-        multiple values with commas. They will be tried
+        More than one value can be specified by separating
+        multiple values with whitespace. They will be tried
         in turn, until one is written without error. If
-        neither suceeds, the operation will be aborted.
+        neither succeeds, the operation will be aborted.
         </para></listitem>
       </varlistentry>
     </variablelist>
@@ -160,10 +160,8 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
     <para>Example: to exploit the <quote>freeze</quote> mode added
     in Linux 3.9, one can use <command>systemctl suspend</command>
     with
-    <programlisting>
-[Sleep]
-SuspendState=freeze
-    </programlisting></para>
+    <programlisting>[Sleep]
+SuspendState=freeze</programlisting></para>
   </refsect1>
 
   <refsect1>
